projects/ccd-case-ui-toolkit/src/lib/shared/components/pagination/pagination.component.html

```diff
@@ -18,7 +18,7 @@
       <li [class.current]="p.getCurrent() === page.value" [class.ellipsis]="page.label === '...'"
         *ngFor="let page of p.pages">
         <a tabindex="0" (keyup.enter)="p.setCurrent(page.value)" (click)="p.setCurrent(page.value)"
-          *ngIf="p.getCurrent() !== page.value">
+          *ngIf="p.getCurrent() !== page.value" [attr.aria-label]="(page.label === '...') ? 'Load more pages' : 'Page ' + (page.label | number:'') | rpxTranslate">
```

Comment:
Can you find a way to avoid sending "page.label | number:" to the translation service. Do send "Load more pages" though. Actually is "load more pages" the correct description for what the ... does?